tst/gputests/TestUtils.h

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
+#include <string>
+
```

Comment:
You should have #includes for vector and anything else you use from the standard library.